pydantic/fields.py

```diff
@@ -209,6 +210,17 @@ def _populate_sub_fields(self) -> None:  # noqa: C901 (ignore complexity)
             return
 
         if issubclass(origin, List):
+            # Create self validators
+            get_validators = getattr(self.type_, '__get_validators__', None)
+            if not get_validators:
+                get_validators = getattr(self.type_, 'get_validators', None)
```

Comment:
You don't need this case, we'll remove support for `get_validators` soon anyway.

Reply:
Done